robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -19,14 +19,16 @@
 % second row of A will have the joints offsets.
 %
 % DQ_SerialManipulatorDH Methods (Concrete):
-%       get_dim_configuration_space - Return the dimension of the configuration space.
-%       fkm - Compute the forward kinematics while taking into account base and end-effector's rigid transformations.
-%       plot - Plots the serial manipulator.
-%       pose_jacobian - Compute the pose Jacobian while taking into account base's and end-effector's rigid transformations.
+%       get_thetas - Returns the first row of the DH Matrix ('theta' parameter). 
+%       get_ds - Returns the second row of the DH Matrix ('d' parameter).
+%       get_as - Returns the third row of the DH Matrix ('a' parameter).
+%       get_alphas - Returns the fourth row of the DH Matrix ('alpha' parameter).
+%       get_types - Returns the actuation type, either DQ_SerialManipulatorDH.JOINT_ROTATIONALor DQ_SerialManipulatorDH.JOINT_PRISMATIC
```

Comment:
I modified the name from JOINT_ROTATIONAL and JOINT_PRISMATIC to REVOLUTE AND PRISMATIC, respectively.
@mmmarinho do you think that "JOINT" is necessary for the name?

Reply:
@juanjqo @bvadorno If MATLAB has named enums, then we can use that instead. If it doesn’t… then let’s think about it a bit. 

In my opinion, just PRISMATIC or just ROTATIONAL lacks meaning. In C++ I would write DQSerialManipulatorDH::JointType::Rotational with an enum class.

If those are not available, having JOINT_ first is a common style in C enums because it didn’t (or still doesn’t, not sure) have typed enums. Then all related enums were grouped together by their prefix, see OpenCV for one quick example
https://docs.opencv.org/3.4/d9/d6a/group__aruco.html.

Reply:
@juanjqo @bvadorno 

I prefer the enumeration class and use it profusely everywhere else. So I'm happy with this change.

Either way, because that old-style enum was available in the last release version (20.04)
https://github.com/dqrobotics/matlab/blob/f11f0de06784d04042b8713010373d08fb86ccf8/robot_modeling/DQ_SerialManipulatorDH.m#L62
we should keep it as deprecated for at least one version, as per our current legacy policy. All examples should be updated to use the new style.

Reply:
@bvadorno @mmmarinho I added again the properties JOINT_ROTATIONAL and JOINT_PRISMATIC.